RELEASE_NOTES.md

```diff
@@ -16,4 +16,4 @@
 
 ## Bug Fixes
 
-<!-- Here goes notable bug fixes that are worth a special mention or explanation -->
+- When the PowerDistributor receives a zero power request for PV inverters, it now correctly sets zero power to the inverters, and no longer crashes.
```

Comment:
Shouldn't it be `... receives a positive or zero power request ...`

Reply:
It would never receive a positive power, because that's filtered in the pv pool: https://github.com/frequenz-floss/frequenz-sdk-python/blob/4b88ab288c95e9acb8b88f66ccc625b661a856b0/src/frequenz/sdk/timeseries/pv_pool/_pv_pool.py#L109-L110